hack/local-up-cluster.sh

```diff
@@ -44,6 +32,57 @@ RoleARN="$NODE_ROLE_ARN"
 EOF
 }
 
-write_cloudconfig
+CLOUD_PROVIDER_ROOT=${CLOUD_PROVIDER_ROOT:-"$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." &> /dev/null && pwd )"}
+KUBE_ROOT=${KUBE_ROOT:-"$GOPATH/src/k8s.io/kubernetes"}
+KUBE_VERSION=${KUBE_VERSION:-v1.22.1}
+
+# Configuration for k8s.io/kubernetes/hack/local-up-cluster.sh
+NODE_ROLE_ARN=${NODE_ROLE_ARN:-""}
+CLOUD_PROVIDER=aws
+EXTERNAL_CLOUD_PROVIDER=true
+CLOUD_CONFIG=${CLOUD_CONFIG:-${CLOUD_PROVIDER_ROOT}/cloudconfig}
+EXTERNAL_CLOUD_PROVIDER_BINARY=${EXTERNAL_CLOUD_PROVIDER_BINARY:-"$GOPATH/src/k8s.io/cloud-provider-aws/aws-cloud-controller-manager"}
+NODE_ZONE=${AWS_NODE_ZONE:-"us-west-2a"}
+NET_PLUGIN="${NET_PLUGIN:-kubenet}"
+CLOUD_CTLRMGR_FLAGS=${CLOUD_CTLRMGR_FLAGS:-"--controllers=*"}
```

Comment:
Not supported yet in upstream

Reply:
remove if not getting into upstream

Reply:
https://github.com/kubernetes/kubernetes/pull/105300